securitycenter/synth.py

```diff
@@ -40,6 +40,6 @@
 # Add templated files
 # ----------------------------------------------------------------------------
 templated_files = common.py_library(unit_cov_level=97, cov_level=100)
-s.move(templated_files)
+s.move(templated_files, excludes=['noxfile.py'])
```

Comment:
Added an exclude for the noxfile.

When I re-ran the generation, I got all the routing header changes.